### PR TITLE
added code for data insertion. + few TODO tasks for Riaz

### PR DESCRIPTION
**Pull request by AnumSheraz**

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -2,23 +2,51 @@
...
+        if exists[0][0] == 0:  # don't add user if already exists (only for development purpose)
+            db.run_query(f"INSERT INTO ACTIONS(action_name) VALUES('{action}')")
+
+    # TODO populate role_actions table
+
+
+def populateSchema(db):
+    is_db_exists = db.run_query_return("show databases;")
+    if 'iot' not in [d[0].lower() for d in is_db_exists]:
+        pass
```
Comment (Rali72) [inline main.py]: This only works if the database already exists, basically it does not create db if it does not exists already

Comment (AnumSheraz) [reply main.py:35]: I think this should be `if 'iot' in`.

Comment (AnumSheraz) [reply main.py]: How come ? isn't this saying that if iot not exists in databases, then create one ? 
or am I missunderstanding

Comment (Rali72) [reply main.py]: it does say, but the problem seems to be with the argument provided to "populateSchema" method which presumes database already exists. (my thought)
see my below comments

File: main.py [hunk 2]
```diff
@@ -2,23 +2,51 @@
...
+        # TODO read the iot_schema.sql file and run it against database
+    else:
+        print('database already exists...')
+
+
 if __name__ == "__main__":
     db_creds = configparser.ConfigParser()
     db_creds.read_file(open(r'db_creds.conf'))
     db_client = DbHandler(db_creds)
```
Comment (Rali72) [inline main.py:61]: the "populateSchema" methods takes "db_client" as argument which I think creates the problem

**Pull request merged**